### PR TITLE
Support input type

### PR DESCRIPTION
**Pull request by sagarvijaygupta**
Support 'number' input type in the crawler
Support 'checkbox' input type in the crawler

Solves : #3 and #4

### CODE REVIEW

**Comment by sagarvijaygupta** [comment]
checkbox_unchecked
checkbox_checked
number_without
number_with

File: collect.py [hunk 1]
```diff
@@ -111,6 +111,10 @@ def do_something(driver, elem_id=None):
             elem.send_keys('prova@email.it')
         elif input_type == 'password':
             elem.send_keys('aMildlyComplexPasswordIn2017')
+        elif input_type == 'checkbox':
```
Comment (marco-c) [inline collect.py]: Since this is the same as a `button` or a `a`, can you move it to `if elem.tag_name in ['button', 'a']:`?

**Comment by marco-c** [comment]
Thanks for the PR!

File: collect.py (hunk 1 above)
Comment (sagarvijaygupta) [reply collect.py]: @marco-c the ```tag_name``` for ```checkbox``` is ```input``` so I don't think that we can move it there.

Comment (marco-c) [reply collect.py]: Right 😄

**Pull request merged**